src/applications/vaos/services/location/index.js

```diff
@@ -197,7 +190,7 @@ export function getFacilityIdFromLocation(location) {
  * @param {String} id A location's fake FHIR id
  */
 export function getSiteIdFromFakeFHIRId(id) {
```

Comment:
Somewhat related, I'm wondering if there might be a better name for this helper, now that we aren't referring to the IDs as "fake ids"

Reply:
Good point, I went ahead and renamed this function to better describe what it's doing.